### PR TITLE
Task26,27,28

### PR DESCRIPTION
**Pull request by Sandhyathota1**
completed Task26 , Task27 ,Task28

### CODE REVIEW

**Comment by vajrang-b** [comment]
SourceCode/2023/Sep2023/Basics/Sandhya/Task19/Task19/project.json
1. File: Main.xaml
```
Issue: Could not load file C:\VSTS_BA1\_work\1\s\SourceCode\2023\Sep2023\Basics\Sandhya\Task19\Task19\Main.xaml. Reason: Cannot create unknown type {http://schemas.uipath.com/workflow/activities}ReadTextFile.

Fix: Looks like there's an issue with the 'ReadTextFile' activity. Check if it exists in your UiPath Studio and if you've imported the necessary package for it. If it's a custom activity, ensure you've added it correctly.

2. Project Analysis:
```
Issue: Cannot perform project analysis because some workflows have errors.

Fix: The error in your Main.xaml file prevents the project from analyzing properly. Rectify the error in the Main.xaml file, and then try to perform the project analysis.

**Comment by vajrang-b** [comment]
SourceCode/2023/Sep2023/Basics/Sandhya/Task26/Task26/project.json
The issue appears to be in your Main.xaml file within the Task26 directory. The error message indicates that there is an unknown type '{http://schemas.uipath.com/workflow/activities}MessageBox'.

This usually suggests that there may be a missing or incorrect namespace reference in your XAML file. One easy fix would be to go through your XAML file and ensure that all activity namespaces are correctly included.

In simpler terms, imagine your code as a book. The error is like a word that the book doesn't recognize. You have to make sure that the book has a reference (like a dictionary) to understand what the word (in this case, 'MessageBox') means.

Secondly, the project analysis cannot be performed due to the errors in your workflows. Fixing the first issue may also solve this one.

Please fix this issue and make a new commit. We got this!

**Comment by vajrang-b** [comment]
SourceCode/2023/Sep2023/Basics/Sandhya/Task27/Testing1/project.json
1. Issue: "An exception was thrown while activating UiPath.Studio.Platform.Contracts.IPlatformWrapper."
   Solution: Looks like there's an issue initializing the IPlatformWrapper. Update your UiPath Studio, or reinstall if the issue persists.

2. Issue: "Could not load file C:\VSTS_BA1\_work\1\s\SourceCode\2023...unknown type '{http://schemas.uipath.com/workflow/activities}ForEachRow'"
   Solution: There seems to be a problem loading Main.xaml. The error points to an unknown type 'ForEachRow'. Check that you've correctly set up 'ForEachRow' activity in UiPath. Reinstall necessary dependencies if required.

3. Issue: "An exception was thrown while activating UiPath.Studio.Platform.Contracts.IPlatformWrapper."
   Solution: Similar to the first one, there's an issue initializing the IPlatformWrapper. Try checking your UiPath Studio version and make updates or reinstall the software, if necessary.

4. Issue: "Cannot perform project analysis because some workflows have errors."
   Solution: A bug in one or more of your workflows is preventing project analysis. Review each workflow for errors, and fix these to make the project work. Be surgical, kill those bugs.

**Comment by vajrang-b** [comment]
SourceCode/2023/Sep2023/Basics/Sandhya/Task28/Task28/project.json
To the developer,

Whilst checking through the `Main.xaml` in your Task28 directory, I've hit a roadblock. Looks like your code is calling a type `{http://schemas.uipath.com/workflow/activities}CommentOut` which the system can't quite decipher. Might be worth revisiting and confirming this `{http://schemas.uipath.com/workflow/activities}CommentOut` type to ensure it's correctly defined and used. 

Additionally, there seems to be a problem with the project analysis because some of the workflows contain errors.  Your task now is to debug your workflows, sort out these errors. Resolving these glitches should make the project analysis smooth sailing.

Keep improving and remember, every bug is a lesson learned. Keep it up.

**Comment by krish952-creator** [comment]
@Sandhyathota1 
Task-23
Code verified.
Task-26
Code verified. Default activity names 
 
image

Task-27
Code verified. Default activity names
 
image

Task-28
Code verified. Default activity names
image
